metadata-ingestion/src/datahub/cli/docker_cli.py

```diff
@@ -245,20 +238,16 @@ def _docker_compose_v2() -> Union[List[str], Literal[False]]:
                 # instead of as a plugin.
                 return ["docker-compose"]
 
-            click.secho(
+            raise DockerComposeVersionError(
```

Comment:
Nice - so do these exceptions still appear as beautifully to the end user?

Reply:
yes - that's what the changes in entrypoints.py are for